src/components/ReportActionItem/ReportPreview.js

```diff
@@ -115,23 +118,34 @@ function ReportPreview(props) {
                     accessibilityRole="button"
                     accessibilityLabel={props.translate('iou.viewDetails')}
                 >
-                    <View style={[styles.flexShrink1]}>
-                        {props.iouReport.hasOutstandingIOU ? (
-                            <Text style={[styles.chatItemMessage, styles.cursorPointer, styles.colorMuted]}>
-                                {lodashGet(message, 'html', props.translate('iou.payerOwesAmount', {payer: managerName, amount: reportAmount}))}
-                            </Text>
-                        ) : (
-                            <View style={[styles.flexRow]}>
-                                <Text style={[styles.chatItemMessage, styles.cursorPointer, styles.colorMuted]}>
-                                    {lodashGet(message, 'html', props.translate('iou.payerSettled', {amount: reportAmount}))}
-                                </Text>
+                    <View style={[styles.iouPreviewBox]}>
```

Comment:
i think we do not need [] inside style - because you are using only one style, same change should be done on line 122, 135 as well